requirements.cachito

```diff
@@ -2,6 +2,14 @@ ironic @ git+https://github.com/openshift/openstack-ironic@7f8f0c909fc9a6c80d6e0
 sushy @ git+https://github.com/openshift/openstack-sushy@48e0e7c88634b1c59772ab7cd51fa59ff3258da0
 
 proliantutils===2.16.3
+python-ironicclient===4.9.0
 python-scciclient===0.17.0
 
 # DEPENDENCIES
```

Comment:
If we actually go down this patch, do you think we can do some sort of a cross-validation against upper-constraints? So that we don't accidentally keep using a long untested version? Maybe only on main to avoid messing up backports.

Reply:
that's doable but we need to discuss how that would influence the maintenance of the requirements, I don't want to change the file too often, and upper-constraints change multiple times per week
consider that we already do such check (manually) a couple of times during each release cycle, we're never at the very latest version unless there are significant changes (major version bump, important bug fixes, and so on)

Reply:
I was thinking maybe every time we advance the hash on main (only), enforce updating this file from upper-constaints? This won't happen frequently, but at least once a release cycle, which is better than nothing.